higher_health/management/commands/create_other_dropdown_field_values.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+
+from higher_health.models import Campus, University
+
+
+class Command(BaseCommand):
+    help = "Create Other dropdown fields"
+
+    def handle(self, *args, **options):
+        university_count = University.objects.exclude(name="Other").count()
+        university, created = University.objects.get_or_create(name="Other")
+
+        university.sort_order = university_count + 1
+        university.save()
+
+        campus_count = Campus.objects.exclude(name="Other").count()
+        campus, created = Campus.objects.get_or_create(
+            name="Other", university=university
+        )
+
+        campus.sort_order = campus_count + 1
```

Comment:
So is the plan here to have every row have a `sort_order` value? This will become an issue if we then decide to add more rows to the database, then the `Other` value won't be at the end anymore.

I originally thought that all other rows would have a null/`0` value, and the `Other` row would have a value of `1`, so that it gets prioritised, or something like that.

Reply:
@rudigiesler that can be the case if you want, or you can run a management command to update db the row. However you wish the order will have the "other" at the end of the queryset so long as the sort order is greater than that of the other rows

Reply:
I think having to remember to run a management command every time we update the list of campuses and universities is error prone, so I think it's better if we have `0` for all rows, and then the `Other` row has a value of `1`.

I would change this management command to just set the sort order to `1` on creation, along with a comment that the default is `0`, and we want other to be at the end, so we're setting it to `1`. Otherwise it's very confusing to anyone who comes across this code why we're assigning the sort order according to a count.